### PR TITLE
feat: Form lable supports left alignment

### PR DESCRIPTION
**Pull request by unsung189**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=1292) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=1292) before we can accept your contribution.<br/><br/>:x: unsung189 [sign now](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=1292)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/ant-design-blazor/ant-design-blazor?pullRequest=1292) it.</sub>

File: components/grid/Col.razor.cs [hunk 1]
```diff
@@ -59,6 +59,8 @@ public partial class Col : AntDomComponentBase
 
         [Parameter]
         public OneOf<int, EmbeddedProperty> Xxl { get; set; }
+        [Parameter]
+        public AntLableAlignType LableAlignType { get; set; } = AntLableAlignType.Right;
```
Comment (ElderJames) [inline components/grid/Col.razor.cs:63]: Please rename the property to `LabelAlign` as well as ant design  react.

**Review by ElderJames — CHANGES_REQUESTED** [review]

File: components/grid/Col.razor.cs (hunk 1 above)
Comment (unsung189) [reply components/grid/Col.razor.cs]: @ElderJames

File: components/form/ColLayoutParam.cs [hunk 2]
```diff
@@ -31,25 +31,25 @@ public class ColLayoutParam
 
         public OneOf<int, EmbeddedProperty> Xxl { get; set; }
 
-        public AntLableAlignType LableAlignType { get; set; } = AntLableAlignType.Right;
+        public AntLableAlignType LableAlign { get; set; } = AntLableAlignType.Right;
```
Comment (ElderJames) [inline components/form/ColLayoutParam.cs:34]: typo `LabelAlign`

**Review by ElderJames — CHANGES_REQUESTED** [review]

File: components/form/ColLayoutParam.cs (hunk 2 above)
Comment (unsung189) [reply components/form/ColLayoutParam.cs]: Sorry for such a low-level error, fixed, please review

**Comment by ElderJames** [comment]
That's fine. But why did you add the property in Col instead FormItem?

**Comment by unsung189** [comment]
That's fine. But why did you add the property in Col instead FormItem?

**Review by ElderJames — APPROVED** [review]

**Pull request merged**